openspec/changes/archive/2026-06-06-fix-docker-etc-passwd/proposal.md

```
@@ -0,0 +1,38 @@
+## Why
+
+The ncps container image ships `/etc/passwd` and `/etc/group` as **absolute symlinks into the nix store** (verified on the built image: `etc/passwd -> /nix/store/…-passwd/etc/passwd`). Recent container runtimes do a securejoin/`openat` on `etc/passwd` while creating the container and **reject an absolute symlink as escaping the rootfs**, so the container never starts:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)